src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -63,6 +63,12 @@ def _get_version_in_range(
                 return version
         return ""
 
+    def _get_asm_supported_revision(self):
+        revisions_cmd = "aks mesh get-revisions -l westus2"
```

Comment:
It would be better to pass the location as a parameter as the test case may not always run in westus2 (e.g., it's set to run in westcentralus in cli runner)